### PR TITLE
Home

### PR DESCRIPTION
**Pull request by clairedelaleau**
Home is done. Please, can you check this.

### CODE REVIEW

File: src/pages/Home/index.jsx [hunk 1]
```diff
@@ -1,23 +1,132 @@
...
+        <Link to="/office">
+          <article className="office">
+            <div className="reservation">
+              <h2>Réservation de bureau</h2>
+            </div>
+            <div className="chooseYourOffice">
+              <p>Choisissez votre bureau</p>
+            </div>
+            <div className="officePicture">
+              <img src={Office} alt={Office} />
```
Comment (HeloiseFerla) [inline src/pages/Home/index.jsx]: You can't put a picture in alt.

File: src/pages/Home/index.jsx [hunk 2]
```diff
@@ -1,23 +1,132 @@
...
+        <Link to="/supplies">
+          <article className="furnitures">
+            <div className="order">
+              <h2>Commande de fournitures</h2>
+            </div>
+            <div className="shop">
+              <p>Ajoutez les articles dont vous avez besoin</p>
+            </div>
+            <div className="picturefurniture">
+              <img src={Furnitures} alt={Furnitures} />
```
Comment (HeloiseFerla) [inline src/pages/Home/index.jsx]: same

File: src/pages/Home/index.jsx [hunk 3]
```diff
@@ -1,23 +1,132 @@
...
+        </Link>
+        <Link to="/lostCard">
+          <article className="lostcard">
+            <div className="card">
+              <h2>Carte perdue</h2>
+            </div>
+            <div className="unactivCard">
+              <p>Désactiver votre carte en cas de perte</p>
+            </div>
+            <div className="pictureCard">
```
Comment (HeloiseFerla) [inline src/pages/Home/index.jsx:120]: same

File: src/pages/Home/index.jsx [hunk 4]
```diff
@@ -1,23 +1,132 @@
...
+            <div className="shop">
+              <p>Ajoutez les articles dont vous avez besoin</p>
+            </div>
+            <div className="picturefurniture">
+              <img src={Furnitures} alt={Furnitures} />
+            </div>
+          </article>
+        </Link>
+        <Link to="/lostCard">
+          <article className="lostcard">
```
Comment (HeloiseFerla) [inline src/pages/Home/index.jsx]: className in CamelCase Plz

File: src/pages/Home/index.jsx [hunk 5]
```diff
@@ -1,23 +1,132 @@
...
+        </Link>
+        <Link to="/supplies">
+          <article className="furnitures">
+            <div className="order">
+              <h2>Commande de fournitures</h2>
+            </div>
+            <div className="shop">
+              <p>Ajoutez les articles dont vous avez besoin</p>
+            </div>
+            <div className="picturefurniture">
```
Comment (HeloiseFerla) [inline src/pages/Home/index.jsx]: ClassName in camelCase plz

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request closed without merging**